web/src/components/Pagination.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <span>
+    <span>
+      {{ pageStart }} - {{ pageEnd }}
+    </span>
+    <v-icon
+      :disabled="prevDisabled"
+      @click="prevPage()"
+    >
+      mdi-chevron-left
+    </v-icon>
+    <v-icon
+      :disabled="maxPage"
```

Comment:
[super minor] And probably `lastPage` instead of max? lol

Reply:
ah yeah makes sense, I wrote all of this during yesterday's hangouts call we had so my prain zoned out lol